libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -86,7 +93,57 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     });
   }
 
+  private getContentMaxHeight(): number {
+    const ionModalWrapper = this.elementRef.nativeElement.parentElement;
```

Comment:
Dunno what's most robust if Ionic changes their structure? But could also be:
```suggestion
    const ionModalWrapper = this.elementRef.nativeElement.closest('.modal-wrapper')
```